### PR TITLE
upgrade curling and allow curling from the docker container to work

### LINKED ISSUE #391: Update curling
The curling lib is out of date, lets update.

It would nice to add in .curling file in the root of the project so that it picks that up by default. So the command line works out of the box with OAuth.

### PR DESCRIPTION
**Pull request by andymckay**
Fixes #391

### CODE REVIEW

**Comment by andymckay** [comment]
So that inside docker:

```
bash-4.1# curling http://solitude:2602/generic/buyer/?id=8
{
  "meta": {
    "headers": {
      "content-length": "341",
...```

File: .curling [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "solitude:2602":
+    {
+        "key": "local-curling-client",
+        "secret": "please change this"
+    }
+}
```
Comment (kumar303) [inline .curling]: that's pretty sweet

File: solitude/settings/base.py [hunk 2]
```diff
@@ -225,7 +225,8 @@
 CLIENT_OAUTH_KEYS = {
     'marketplace': 'please change this',
     'webpay': 'please change this',
-    'payments-service': 'please change this'
+    'payments-service': 'please change this',
+    'local-curling-client': 'please change this'
```
Comment (kumar303) [inline solitude/settings/base.py]: trailing comma might be nice

**Comment by kumar303** [comment]
r+wc

**Pull request merged**